mlblocks_primitives/keras.Sequential.LSTMTimeSeriesRegressor.json

```diff
@@ -0,0 +1,122 @@
+{
+    "name": "keras.Sequential.LSTMTimeSeriesRegressor",
+    "author": "Ihssan Tinawi <itinawi@mit.edu>",
+    "documentation": "",
+    "description": "This primitive consists of multiple Keras layers that can pass time-series data through an LSTM in order to predict the value at x_{t+1}",
+    "classifiers": {
+        "type": "estimator",
+        "subtype": "regressor"
+    },
+    "modalities": [],
+    "primitive": "mlprimitives.adapters.keras.Sequential",
+    "fit": {
+        "method": "fit",
+        "args": [
+            {
+                "name": "X",
+                "type": "ndarray"
+            },
+            {
+                "name": "y",
+                "type": "array"
+            }
+        ]
+    },
+    "produce": {
+        "method": "predict",
+        "args": [
+            {
+                "name": "X",
+                "type": "ndarray"
+            }
+        ],
+        "output": [
+            {
+                "name": "y",
+                "type": "array"
+            }
+        ]
+    },
+    "hyperparameters": {
+        "fixed": {
+            "input_length": {
+                "type": "int",
+                "default": 1500
+            },
+            "dense_units": {
+                "type": "int",
+                "description": "Number of classes"
```

Comment:
Since at the end this has to stay but it won't be a classification, can we update this description?
I think that something like "Number of steps ahead to predict" would be more appropriate.
Also, I think that it could be reasonable to put a default value of 1, if that's the expected behavior.